docs/source/Introduction/install.rst

```diff
@@ -6,12 +6,13 @@ Laigter is available for Linux, Windows, and MacOS. Mac version is experimental
 Installing on Windows
 ---------------------
 
-The only way currently to get an installer for Windows, is through `itch.io <https://itch.io/>`_.
+The only ways currently to get an installer for Windows, is through `itch.io <https://itch.io/>`_ .
```

Comment:
Sure it is ways instead of way?

Reply:
Nope, that's an error, I made a first version where I stated that you could also use the windows app in that same sentence but then moved it to the bottom of the section. I forgot to change it back... I'll edit it right away

Reply:
changed